cmake/catkin_package.cmake

```diff
@@ -75,6 +77,7 @@
 #     LIBRARIES projlib1 projlib2
 #     CATKIN_DEPENDS roscpp
 #     DEPENDS Eigen
+#     DEFINES MYDEFINE=1
```

Comment:
These are not compile options, but definitions. `-D` is added automatically: https://github.com/ipa-mdl/catkin/blob/compile-options/cmake/catkin_package.cmake#L385
(I have adopted the syntax from https://github.com/ros/catkin/issues/610)